webScraping/cfeScrapper.js

```diff
@@ -97,23 +103,49 @@ Scrapper.prototype.doScraping = async function () {
       // eslint-disable-next-line prettier/prettier
       this.printPercentage,
     );
-    console.log('Obtained data: ', data.length);
+    console.log(`Obtained data: ${data.length}`);
 
     const object = await data.map((item) => this.createObject(item));
 
-    console.log('Saving data...');
+    console.log(`Saving data...`);
     await myPage.saveFile(object, route);
-    console.log('Data saved in: ', route);
+    console.log(`Data saved in: ${route}`);
 
     await myPage.closeBrowser();
-    console.log('Browser closed successfully');
+    console.log(`Browser closed successfully`);
 
     return;
   } catch (err) {
-    console.error('Error: ', err);
+    console.error(`Error: ${err}`);
     throw err;
   }
 };
 
-// eslint-disable-next-line object-curly-spacing
+/**
+ * Compares the date´s file and today to check if doScraping or not
+ */
+// eslint-disable-next-line space-before-function-paren
+Scrapper.prototype.mainFunction = function () {
```

Comment:
Do not think we need this function, is a particular condition to do or not do the webscraping, is not a feature related with the scraper lib. the execution is correct. Explain to me why should live here.